deploy/olm-catalog/kogito-operator/kogito-operator.package.yaml

```diff
@@ -1,7 +1,13 @@
 channels:
-- currentCSV: kogito-operator.v0.13.0
+- currentCSV: kogito-operator.v1.0.0-rc1
   name: alpha
-- currentCSV: kogito-operator.v0.12.0
+- currentCSV: kogito-operator.v0.13.0
   name: dev-preview
+- currentCSV: kogito-operator.v0.9.1
+  name: original_0_9_0
+- currentCSV: kogito-operator.v0.8.0-rc1
+  name: original_0_8_0
+- currentCSV: kogito-operator.v0.7.0-rc4
+  name: original_0_7_0
```

Comment:
These versions are needed by the OLM manifests since it's not support skipping versions. For example: I have installed from `dev-preview` channel version 0.9.1, I can't upgrade using the same channel to `0.12.0` since the CSV manifests from 0.12.0 says `replaces: 0.11.0`. If we would release a new dev preview, we should also create a new channel for that and instruct users to install from the scratch using the new channel.

I'll update the current dev-preview channel to 0.13.0 since we have lots of bug fixing in this version that's pretty important to everyone.